Makefile

```diff
@@ -46,11 +41,49 @@ dev-dependencies: ## Downloads the necessary dev dependencies.
 	@if [[ "$$(uname)" == 'Darwin' ]]; then brew install semgrep; fi
 .PHONY: dev-dependencies
 
+tidy: ## Cleans the Go module.
+	@echo "==> Tidying module"
+	@$(GO) mod tidy
+.PHONY: tidy
+
+generate: ## Builds and runs generators, to generate code for API endpoints and tests.
+	@echo "==> Removing existing generated code"
+	@rm -f fastly/products/*/api.go fastly/products/*/api_test.go
+	@echo "==> Building generators"
+	@$(GO) build -o generators ./internal/generators/...
```

Comment:
Why build first? Can this just be go run?

Second, could these be embedded as go:generate lines?

Reply:
Using `go run` in the `go:generate` lines would require that those lines have the relative path to the generator source code... that seemed a bit icky and fragile to me. This method produces binaries that are temporarily in the `PATH`, so the `go:generate` lines only need to include the name of the desired generator.